Makefile

```diff
@@ -27,6 +27,16 @@ clean:
 tests/generated: json
 	go generate ./tests
 
+# build builds the cmd/wit-bindgen-go binary
+.PHONY: build
+build: internal/wasmtools/wasm-tools.wasm
+	go build -o wit-bindgen-go ./cmd/wit-bindgen-go
+
+internal/wasmtools/wasm-tools.wasm:
+	cd internal/wasmtools && \
+	cargo build --target wasm32-wasip1 --release -p wasm-tools
```

Comment:
Can we use `--target-dir` to write the wasm-tools.wasm binary directly into the dir?

Reply:
--target-dir will put all the generated artifacts, including the binary, into that dir, which is probably not what we want.